### PR TITLE
Added App Pages

### PR DESCRIPTION
**Pull request by BohdanBodakva**

### CODE REVIEW

File: lib/elements/footer.dart [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+class _FooterState extends State<Footer> {
+  
+
+  @override
+  Widget build(BuildContext context) {
+    final mediaQuery = MediaQuery.of(context);
+
+    return Container(
+      // width: _mediaQuery.size.width,
```
Comment (DenysDoskochynskyi) [inline lib/elements/footer.dart]: remove comments

File: lib/elements/footer.dart [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  @override
+  State<Footer> createState() => _FooterState();
+}
+
+class _FooterState extends State<Footer> {
+  
+
+  @override
+  Widget build(BuildContext context) {
+    final mediaQuery = MediaQuery.of(context);
```
Comment (DenysDoskochynskyi) [inline lib/elements/footer.dart]: better to use sizeOf(it's a little bit more optimised)
Please remember me to tell you why(when you would demo your work )

File: lib/enums/font_size.dart [hunk 3]
```diff
@@ -0,0 +1,22 @@
+// import 'package:flutter/material.dart';
+
+import 'package:flutter/material.dart';
+
+class MyFontSize{
+
+  static double getFontSize(BuildContext context, int i){
+    final width = MediaQuery.of(context).size.width;
```
Comment (DenysDoskochynskyi) [inline lib/enums/font_size.dart]: Please replace a `MediaQuery.of(context).size.width;` to `MediaQuery.sizeOf(context).width;`

File: lib/enums/font_size.dart [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    switch(i){
+      case 1: return width * 0.035;
+      case 2: return width * 0.045;
+      case 3: return width * 0.055;
+      case 4: return width * 0.065;
+      case 5: return width * 0.075;
+      case 6: return width * 0.085;
+    }
+    
+    return -1;
```
Comment (DenysDoskochynskyi) [inline lib/enums/font_size.dart]: you can use for that just a `default: return -1;`

File: lib/main.dart [hunk 5]
```diff
@@ -1,154 +1,64 @@
 import 'package:flutter/material.dart';
+import 'package:my_flutter_application/elements/my_calendar.dart';
+import 'package:my_flutter_application/pages/home.dart';
+// import 'package:my_flutter_application/logic/storage.dart';
```
Comment (DenysDoskochynskyi) [inline lib/main.dart]: comments

File: lib/elements/homework_indicator.dart
Comment (DenysDoskochynskyi) [inline lib/elements/homework_indicator.dart:1]: remove unused code

File: lib/elements/info_dialog.dart [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+                    children: widgetList,
+                  ),
+                ), 
+              ),
+            ),
+            widget.saveButton,
+          ],
+      ),
+      ),  
+      actions: <Widget>[
```
Comment (DenysDoskochynskyi) [inline lib/elements/info_dialog.dart]: removed `<Widget>`

**Review by DenysDoskochynskyi — COMMENTED** [review]
fix comments :(

**Comment by BohdanBodakva** [comment]
Fixed all coments. Screenshots of app work are attached.
Screenshot_1
Screenshot_2
Screenshot_3
Screenshot_4
Screenshot_5

File: lib/elements/my_lesson_box.dart [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+                  ),
+                ],
+              ),
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.only(top: 14, bottom: 14, right: 10),
+            child: Container(
+              width: 1.5,
+              color: const Color.fromARGB(255, 20, 111, 185),
```
Comment (DenysDoskochynskyi) [inline lib/elements/my_lesson_box.dart]: Will be nice to make as a const reusable variable

File: lib/elements/my_lesson_box.dart [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.only(top: 14, bottom: 14),
```
Comment (DenysDoskochynskyi) [inline lib/elements/my_lesson_box.dart]: You can use EdgeInsets.symetric(vertical:14)

**Comment by BohdanBodakva** [comment]
Fixed all commnets/remarks, thanks.

**Pull request merged**